test/integration/user_can_view_items_test.rb

```diff
@@ -0,0 +1,14 @@
+require 'test_helper'
+
+class UserCanViewAllItemsTest < ActionDispatch::IntegrationTest
+  test "displays all items" do
+    item_1, item_2 = create_list(:item, 2)
+
+    visit '/items'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.